docs/_data/portland-2022-config.yaml

```diff
@@ -230,7 +231,8 @@ flagticketsonsale: True
 flagsoldout: False
 flagspeakersannounced: True
 flagrunofshow: False
-flaghasschedule: False
+flaghasschedule: True
```

Comment:
```suggestion
flaghasschedule: False
```

Although for the build not to break, I also have to turn the schedule off?

Reply:
Correct. If you are building a schedule, it must be complete, except if the incomplete flag is set. This may not be the most optimal config, but I wanted a solution fast before we ended up with a dozen google sheets again ;)